client/src/components/app-mount-notifier.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { render, waitFor } from '@testing-library/react';
+import { Provider } from 'react-redux';
+
+import { i18nextCodes } from '../../../config/i18n/all-langs';
+import AppMountNotifier from './app-mount-notifier';
+import { createStore } from '../redux/createStore';
+import i18nTestConfig from '../../i18n/configForTests';
+
+jest.mock('react-ga');
+jest.unmock('react-i18next');
+
+type Language = [string, string];
+
+const store = createStore();
+
+// Create a nested array for languages
+const languages = Object.keys(i18nextCodes).map(
+  /* eslint-disable @typescript-eslint/ban-ts-comment, @typescript-eslint/no-unsafe-return */
+  // @ts-ignore
+  // TODO: convert `all-langs.js` to TypeScript
+  (key): Language => [i18nextCodes[key], key]
+);
```

Comment:
This is just something fancy to make our tests nicer. What this does is that it converts the `i18nextCodes` object to a nested array for languages, which would then be loop through in the tests. This is a one-time setup that will cover new languages in the future as we are expanding our localization effort.

```js
// Before
const i18nextCodes = {
  english: 'en',
  espanol: 'es',
  // ...
};

// After
const languages = [
  ['en', 'english'],
  ['es', 'espanol'],
  // ...
]
```